### DISCUSSION TITLE
Application does not have access to one or more requested data elements

### DISCUSSION THREAD

**Discussion by syedjamil12** [abuzuhri/Amazon-SP-API-CSharp#207] [Q&A]
Hello dear,

I am trying to getting vendor central fulfillment orders but getting this exception, **please help me in this regards.**
{
  "errors": [
    {
      "message": "Access to requested resource is denied.",
     "code": "Unauthorized",
     "details": ""
    }
  ]
}

**Comment by abuzuhri** [comment]
Since I don’t have access key for vendor all methods not tested , the best way here install code after folk my library debug and fix the problem if there and then push your changes or fixes to library .


Because you start with that open case in Amazon seller center as why you get this error and give them all details (maybe you have access for direct fulfilment and and try to access normal order
